clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentWgsUbam.scala

```diff
@@ -17,36 +17,36 @@ case class DocumentWgsUbam(
   lane: Int,
   libraryName: String,
   location: Location,
-  analysisType: Option[String] = None,
-  baitIntervals: Option[String] = None,
-  baitSet: Option[String] = None,
-  dataType: Option[String] = None,
+  analysisType: Option[Symbol] = None,
+  baitIntervals: Option[Symbol] = None,
+  baitSet: Option[Symbol] = None,
+  dataType: Option[Symbol] = None,
   individualAlias: Option[String] = None,
   initiative: Option[String] = None,
-  lcSet: Option[String] = None,
+  lcSet: Option[Symbol] = None,
   libraryType: Option[String] = None,
-  machineName: Option[String] = None,
-  molecularBarcodeName: Option[String] = None,
-  molecularBarcodeSequence: Option[String] = None,
+  machineName: Option[Symbol] = None,
+  molecularBarcodeName: Option[Symbol] = None,
+  molecularBarcodeSequence: Option[Symbol] = None,
   pairedRun: Option[Boolean] = None,
-  productFamily: Option[String] = None,
-  productName: Option[String] = None,
-  productOrderId: Option[String] = None,
-  productPartNumber: Option[String] = None,
-  project: Option[String] = None,
-  readStructure: Option[String] = None,
+  productFamily: Option[Symbol] = None,
+  productName: Option[Symbol] = None,
+  productOrderId: Option[Symbol] = None,
+  productPartNumber: Option[Symbol] = None,
+  project: Option[Symbol] = None,
```

Comment:
I talked with KT and Charlotte about the mismatch between `Symbol` for project in ubams and `String` for project in gvcfs / crams. Ops has some use-cases where it's helpful to wildcard-match project, so I think this should stay as `String`. `researchProjectId` should also stay as `String` by extension (because a cram `project` can come from either a `project` or `researchProjectId` on the ubam side).